spec/models/post_spec.rb

```diff
@@ -0,0 +1,33 @@
+require 'rails_helper'
+
+RSpec.describe Post, type: :model do
+  first_user = User.create(name: 'Tom', photo: 'Tom.png', bio: 'Teacher from Mexico.')
+  subject { Post.create(author: first_user, title: 'Hello', text: 'This is my first post') }
+
```

Comment:
- [ ] Kindly add a test for the `update_posts_counter` method you have in your Post model as it is required to test all the custom methods in each model.

Reply:
- [ ] Please kindly make this change suggested by the previous reviewer. `update_posts_counter` is not a private method so you should be able to create a test for it.